README.md

```diff
@@ -87,10 +87,14 @@ Now you are free to hack on force-bridge.
 
 ### Troubleshooting `yarn start`
 
-If yarn start fails you can't really know why, so here are the biggest 3 things:
+If yarn start fails, there is no error telling you why, only that it failed. So here are the biggest 3 things:
```

Comment:
If these are your revisions, they need to be added to a different commit as they don't correspond to 'added ada bridge fees'

Reply:
Yeah I'm not sure why those came in, I think I may have accidentally rebased one commit too far. I will fix this / take it out